src/main/java/com/prototype/booking/api/BookRoomApp.java

```diff
@@ -0,0 +1,92 @@
+package com.prototype.booking.api;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Data
+@Builder
+@NoArgsConstructor @AllArgsConstructor
+@Entity(name="BOOK_ROOM_APP")
+public class BookRoomApp {
```

Comment:
As this is recording a booking request for a meeting room, give a closer name. it says App.
maybe like **MeetingRoomRequest**, **MeetingRoomBooking** etc

Reply:
MeetingRoomBooking should be good since it is the application that created when user requests to book room.